SPECS/libsolv/libsolv.spec

```
@@ -1,10 +1,11 @@
 Summary:        A free package dependency solver
 Name:           libsolv
-Version:        0.7.24
+Version:        0.7.28
 Release:        1%{?dist}
 License:        BSD
 URL:            https://github.com/openSUSE/libsolv
-Source0:        https://github.com/openSUSE/libsolv/archive/%{version}/%{name}-%{version}.tar.gz
+#Source0:        https://github.com/openSUSE/libsolv/archive/refs/tags/%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
```

To resolve the URL path vs actual package name, please use this approach:
```suggestion
Source0:        https://github.com/openSUSE/libsolv/archive/refs/tags/%{version}.tar.gz#/%{name}-%{version}.tar.gz
```

It's needed by our automation to work correctly, if I remember correctly.